Assignment/solution_student/css/style.css

```diff
@@ -19,31 +20,229 @@ html {
 .clearfix {
   *zoom: 1;
 }
+
 .clearfix:before,
 .clearfix:after {
     content: " ";
     display: table;
 }
+
 .clearfix:after {
   clear: both;
 }
+
 /******************************************
 /* BASE STYLES
 /*******************************************/
+
 body {
+  width: 50%;
+  opacity: .85;
   color: black;
+  margin: 0 auto;
   font-size: 16px;
-  background: white;
+  font-family: 'open sans', sans-serif;
 }
 
-/******************************************
-/* LAYOUT
-/*******************************************/
+header {
+	width: 100%;
+	background-color: #033048
+}
 
-/******************************************
-/* PAGES
-/*******************************************/
+nav {
+	float: right;
+	margin-top: 20px;
+	padding-right: 50px;
+}
 
-/******************************************
-/* SECTIONS
-/*******************************************/
\ No newline at end of file
+ul {
+    margin: 0;
+    padding: 0;
+    list-style-type: none;
+}
+
+li {
+	display: inline;
+}
+
+h1 {
+	margin: 0;
+	float: left;
+	color: #f0efef;
+	font-size: 1.6em; /*I'm not really sure why I have to use 'em' here. Can you help explain when and why this is needed (versus px)?*/
```

Comment:
Oh, did it say somewhere specifically to use ems here, or you were you just experimenting?  It's really more a matter of preference rather than there are situations where you would _need_ to use one over another. An em is a relative unit of measurement that is based on the nearest specified font-size, which could be the same selector itself, or the parent element, or the `body` itself. So in your case, 1.6ems == 25.6px, because the 1.6 is a relative unit of the body font-size, which is the default 16px (1.6 x 16 = 25.6), which you can see if you inspect the `h1` and look at the 'computed' tab: http://d.pr/i/1iDJe It is a bit confusing, which is why I kinda avoid explicitly teaching it, especially when you can do it with px. Here's an article talking about why you might use it: https://css-tricks.com/why-ems/, but personally, I declare all of my font-sizes these days in px's, and use ems specifically for `line-height`, _because_, keeping track of font-sizes in ems becomes tedious because of it's cascading property, _but_ because it cascades and is based on the nearest font-size, it is good for keeping line-heights proportional when changing font-sizes at different screen sizes, so once we start getting into responsive design, we might want to make our font-sizes smaller or larger depending on the screen size the site is being viewed on, but if we've consciously established a vertical rhythm with line-height in our paragraphs and headings, that vertical rhythm should be proportional to the font-size change, so while I might need to change my font-size at different break points, I should only have to declare the line-height once if I've used ems, because it is relative to the font size — I think that sounded super confusing, and if it di, don't worry about it, but if you want me to sit down with you to show you what I'm talking about, I'd be happy to!